data/src/main/java/org/oppia/data/backends/gae/model/AdditionalStorySummary.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class AdditionalStorySummary(
+
+  @Json(name = "id") val id: String?,
```

Comment:
Are there any differences between this & the canonical version? Are they also different structures in the backend? It might be nice to consolidate them into 1 object if not, since they seem to be the same structure with different uses.

Reply:
Merged Additional and Canonical into "StorySummary" model.